packages/mesos/extra/dcos-mesos-slave.service

```diff
@@ -12,6 +12,7 @@ EnvironmentFile=/opt/mesosphere/environment
 EnvironmentFile=/opt/mesosphere/etc/mesos-slave-common
 EnvironmentFile=-/opt/mesosphere/etc/mesos-slave-common-extras
 EnvironmentFile=/opt/mesosphere/etc/mesos-slave
+EnvironmentFile=-/opt/mesosphere/etc/mesos-slave-extras
```

Comment:
My point is I don't see any usage which we'll have of mesos-slave-extras / mesos-slave-public-extras. The mesos-slave / mesos-slave-public environment files just contain the differences in resources (default role). Modifying that downstream doesn't really make sense to me.